### PR TITLE
More Ore Dictionary support for recipes

### PR DESCRIPTION
**Pull request by Adaptivity**
Chests and crafting tables were registered in Ore Dictionary to support Wood Stuff mod by ganymedes01. I've decided to make a PR for 7.0.x branch instead of 7.1.x since 7.1.x still in Beta and there are no ETA when it's out from Beta. I've tested this in-game and didn't found any issues, so it's shouldn't hurt anything. (I know very little about coding mods, so I'm not sure that I've put OreDictionary.registerOre strings into correct place but at least it works)

### CODE REVIEW

File: common/buildcraft/BuildCraftTransport.java [hunk 1]
```diff
@@ -676,11 +676,11 @@ public void loadRecipes() {
 				String dye = ColorUtils.getOreDictionaryName(15 - i);
 				GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(lensItem, 8, i),
 						"OSO", "SGS", "OSO",
-						'O', "ingotIron", 'S', dye, 'G', "blockGlass"
+						'O', "ingotIron", 'S', "dye", 'G', "blockGlass"
 						));
 				GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(lensItem, 8, i + 16),
 						"OSO", "SGS", "OSO",
-						'O', Blocks.iron_bars, 'S', dye, 'G', "blockGlass"
+						'O', Blocks.iron_bars, 'S', "dye", 'G', "blockGlass"
```
Comment (AEnterprise) [inline common/buildcraft/BuildCraftTransport.java]: if you change this then get it out of the for loop as you are now registering the same recipe multiple times

File: common/buildcraft/BuildCraftCore.java [hunk 2]
```diff
@@ -364,6 +364,9 @@ public void loadConfiguration(FMLPreInitializationEvent evt) {
 		FMLCommonHandler.instance().bus().register(this);
 		MinecraftForge.EVENT_BUS.register(this);
 		MinecraftForge.EVENT_BUS.register(new BlockHighlightHandler());
+
+		OreDictionary.registerOre("chestWood", Blocks.chest);
+		OreDictionary.registerOre("craftingTableWood", Blocks.crafting_table);
```
Comment (AEnterprise) [inline common/buildcraft/BuildCraftCore.java]: maybe check if it isn't registered yet? not sure what the effect of double entries would be

Comment (Adaptivity) [inline common/buildcraft/BuildCraftCore.java]: Double entries makes no effect. It's safe to use just OreDictionary.registerOre.

File: common/buildcraft/BuildCraftTransport.java (hunk 1 above)
Comment (Adaptivity) [inline common/buildcraft/BuildCraftTransport.java]: Should I just remove "for (int i = 0; i < 16; i++) {String dye = ColorUtils.getOreDictionaryName(15 - i);"?

Comment (AEnterprise) [inline common/buildcraft/BuildCraftTransport.java]: on second look: revert this change, these recipes are for making differently colored lenses

**Comment by Adaptivity** [comment]
I hope @asiekierka can merge two commits because Github for Windows very derpy thing and doesn't allow me to squash commits. Or I'll create another PR :)
